Sources/ArgumentParser/Parsable Types/ExpressibleByArgument.swift

```diff
@@ -22,7 +22,9 @@ public protocol ExpressibleByArgument {
   /// An array of all possible strings that can convert to a value of this
   /// type.
```

Comment:
Let's make clear that this is for the help screen, not a parsing requirement.

```suggestion
  /// An array of all possible strings that can convert to a value of this
  /// type, for display in the help screen.
```